### PR TITLE
aufile: add methods to get size in bytes/length in ms

### PR DESCRIPTION
**Pull request by larsimmisch**
Create two new functions:
- `aufile_get_size()`
- `aufile_get_length()`

The former returns the size in bytes, the latter the length in milliseconds.

### CODE REVIEW

File: rem/aufile/aufile.c [hunk 1]
```diff
@@ -228,3 +228,49 @@ int aufile_write(struct aufile *af, const uint8_t *p, size_t sz)
...
+ * Get size of a WAV file in bytes
+ *
+ * @param af  Audio-file
+ *
+ * @return size in bytes if success, otherwise EINVAL
+ */
+size_t aufile_get_size(struct aufile *af)
+{
+	if (!af)
+		return EINVAL;
```
Comment (sreimers) [inline rem/aufile/aufile.c:242]: Better to return `0` on error.

**Comment by sreimers** [comment]
Thanks for PR, looks good, only the return err cases should be better `0`.

File: rem/aufile/aufile.c (hunk 1 above)
Comment (larsimmisch) [reply rem/aufile/aufile.c]: Oh yes!

File: rem/aufile/aufile.c [hunk 2]
```diff
@@ -228,3 +228,53 @@ int aufile_write(struct aufile *af, const uint8_t *p, size_t sz)
...
+size_t aufile_get_size(struct aufile *af)
+{
+	if (!af)
+		return 0;
+
+	return af->datasize;
+}
+
+/**
+ * Get size of a WAV file in bytes
```
Comment (sreimers) [inline rem/aufile/aufile.c:248]:
```suggestion
 * Get length of a WAV file in ms
```

**Pull request merged**